File: rest/short-codes/instance-get-example-1/instance-get-example-1.4.x.cs
// Download the twilio-csharp library from twilio.com/docs/csharp/install
using System;
using Twilio;
class Example 
{
  static void Main(string[] args) 
  {
    // Find your Account Sid and Auth Token at twilio.com/user/account
    string AccountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
    string AuthToken = "your_auth_token";
    var twilio = new TwilioRestClient(AccountSid, AuthToken);

    var shortCode = twilio.GetShortCode("SC6b20cb705c1e8f00210049b20b70fce2");
    
    Console.WriteLine(shortCode.ShortCode);
  }
}

File: rest/authorized-connect-apps/instance-get-example-1/instance-get-example-1.4.x.php
<?php
// Get the PHP helper library from https://twilio.com/docs/libraries/php
require_once('/path/to/twilio-php/Services/Twilio.php'); // Loads the library

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"; 
$token = "your_auth_token"; 
$client = new Services_Twilio($sid, $token);

// Get an object from its sid. If you do not have a sid,
// check out the list resource examples on this page
$authorized_app = $client->account->authorized_connect_apps->get("CN47260e643654388faabe8aaa18ea6756");
echo $authorized_app->connect_app_homepage_url;
